url-updater/src/main/java/com/devonfw/tools/ide/url/updater/docker/DockerDesktopUrlUpdater.java

```diff
@@ -33,8 +33,8 @@ protected void addVersion(UrlVersion urlVersion) {
       if (!success) {
         return;
       }
-      doAddVersion(urlVersion, "https://desktop.docker.com/mac/main/amd64/" + code + "/Docker.dmg", MAC, X64);
-      doAddVersion(urlVersion, "https://desktop.docker.com/mac/main/arm64/" + code + "/Docker.dmg", MAC, ARM64);
+      doAddVersion(urlVersion, "https://desktop.docker.com/mac/main/amd64/" + code + "/Docker.dmg", MAC, X64, "");
```

Comment:
in such case it is better to keep a method with the exact old signature that internally delegates to the new one with the added parameter.

Reply:
Please address in "refactoring PR". The need to pass an empty checksum for all cases where an architecture is specified seems pointless.